plenum/server/pool_manager.py

```diff
@@ -325,20 +325,7 @@ def getNodeName(self, nym):
         return nodeTxn[DATA][ALIAS]
 
     def doStaticValidation(self, identifier, reqId, operation):
-        if operation[TXN_TYPE] == NODE:
-            if not (DATA in operation and isinstance(operation[DATA], dict)):
-                error = "'{}' is missed or not a dict".format(DATA)
-                raise InvalidClientRequest(identifier, reqId, error)
-            # VerKey must be base58
-            if len(set(operation[TARGET_NYM]) - set(base58.alphabet)) != 0:
-                error = "'{}' is not a base58 string".format(TARGET_NYM)
-                raise InvalidClientRequest(identifier, reqId, error)
-
-            data = operation[DATA]
-            for fn, validator in self._dataFieldsValidators:
-                if fn in data and not validator(data[fn]):
-                    error = "'{}' ('{}') is invalid".format(fn, data[fn])
-                    raise InvalidClientRequest(identifier, reqId, error)
+        pass
```

Comment:
This code were added before our validation implemented and duplicate it. Since now our validation is performed before any other checks this is not needed any more.
I discussed it with Alexander and it looks like it was planned the same way